src/components/data/FileBrowser.js

```diff
@@ -73,19 +79,26 @@ export const FileBrowserPanel = _.flow(
 
   // Lifecycle
   useEffect(() => {
+    // Whenever the prefix changes, reload the table right away
     load(prefix)
-  }, [prefix, uploadStatus]) // eslint-disable-line react-hooks/exhaustive-deps
+  }, [prefix, lastRefresh]) // eslint-disable-line react-hooks/exhaustive-deps
 
   useEffect(() => {
-    // Count all the files in the bucket any time the uploader completes
     if (!uploadStatus.active) {
+      // If the uploader has completed, count all the files in the bucket
       count()
+    } else {
+      // While the uploader is working, refresh the table no more often than every 5 seconds
```

Comment:
Yep, I tried both and left some inconsistency in there. Sorry. Switched back to 5, since you're right, it provides the user with more immediate feedback.